File: protect/src/main/java/com/niu/protect/tools/ILog.java
package com.niu.protect.tools;

import android.text.TextUtils;
import android.util.Log;
public class ILog {
    private static final boolean isDebug = true;

    public static void log(Object obj) {
        if (obj != null) {
            Log.i("ILOG", obj.toString());
        }
    }

    public static void d(String tag, String msg) {
        if (!TextUtils.isEmpty(msg)) {
            Log.d(tag, msg);
        }
    }
}

File: corelib/src/main/java/im/niu/corelib/utils/ILog.kt
package im.niu.corelib.utils

import android.text.TextUtils
import android.util.Log

object ILog {
    private const val isDebug = true

    fun log(obj: Any?) {
        if (obj != null) {
            Log.i("ILOG", obj.toString())
        }
    }

    fun d(tag: String?, msg: String?) {
        if (!TextUtils.isEmpty(msg)) {
            Log.d(tag, msg!!)
        }
    }
    fun e(tag: String?, msg: String?) {
        if (!TextUtils.isEmpty(msg)) {
            Log.e(tag, msg!!)
        }
    }
    fun i(tag: String?, msg: String?) {
        if (!TextUtils.isEmpty(msg)) {
            Log.i(tag, msg!!)
        }
    }
}